src/open_inwoner/conf/parts/maileditor.py

```diff
@@ -410,6 +411,28 @@ def _readfile(name):
                 "name": "question",
```

Comment:
@Bartvaderkin We do not want the question itself to actually become visible in the email, but we do need the subject and perhaps the question number.